src/app/examples/floating-action-button-example/floating-action-button-example.component.html

```diff
@@ -13,7 +13,3 @@
 <kirby-floating-action-button>
     <kirby-icon name="close"></kirby-icon>
 </kirby-floating-action-button>
```

Comment:
Please also add example with `colortype="secondary"`

Reply:
we don't have possibility to change FAB color in current version